examples/learn-example/components/DataRichDocument.tsx

```diff
@@ -7,14 +7,12 @@ import { Mermaid } from '@flowershow/core';
 // to handle import statements. Instead, you must include components in scope
 // here.
 const components = {
-  Table: dynamic(() => import('./Table')),
+  Table: dynamic(() => import('@portaljs/components').then(mod => mod.Table)),
```

Comment:
Could you explain what this `.then(mod => mod.Table)` does?

Reply:
It's the same as with `Table: dynamic(() => import('./Table'))`, but since it's a package we have to use the `then` and specify which export we want to import.